boggle_blast.lic

```diff
@@ -0,0 +1,165 @@
+      wait_for_script_to_complete('safe-room')
+      pause
+      walk_to(@portal_room)
+      bput("go path", "Obvious paths: none")
+      pause 5 # assumes quick spider travel option purchased
+      walk_to('10584')
+      pause
+    end
+  end
+
+  def check_money
+    get_money if check_wealth('dokoras') < 200 && @withdraw
+  end
+
+  def get_money
+    walk_to('10358')
+    case bput("withdraw #@cash #@denomination #@currency ", 'The clerk counts out', 'The clerk tells you', 'You must be at')
+    when "The clerk counts out"
+      echo "got cash"
+      return
+    else
+      echo "Out of cash!"
+      exit
+    end
+  end
+
+  def go_play
+    walk_to('10584')
+    unless bput('give farmer 200 dokoras', 'The farmer takes your money', 'Sorry, the game rooms are all full right now') == "The farmer takes your money"
+      while line = get
+        if line =~ /^The pumpkin farmer (beams happily|grins) at/
+          response = bput('give farmer 200 dokoras', 'The farmer takes your money', 'Sorry, the game rooms are all full right now')
+          next if response =~ /Sorry, the game rooms are all full right now/
+          break if response =~ /The farmer takes your money/
+        end
+      end
+    end
+    5.times do
+      @true_pumpkin = false
+      get_pumpkin while !@true_pumpkin
+      case bput('load slingshot', 'You carefully load', "You need to have a pumpkin")
+      when "You need to have a pumpkin"
+        stow_hands
+        @true_pumpkin = false
+        get_pumpkin while !@true_pumpkin
+        bput('load slingshot', 'You carefully load')
+      end
+      case bput('fire slingshot', 'You position the slingshot', 'You let loose', 'You carefully aim the', 'You really should be standing to play', 'You fire the slingshot and awkwardly', 'You take careful aim')
+      when 'You really should be standing to play'
+        bput('stand', 'You stand back up')
+        bput('fire slingshot', 'You position the slingshot', 'You let loose', 'You carefully aim the', 'You fire the slingshot and awkwardly', 'You take careful aim')
+      end
+      pause
+    end
+    pause
+    case bput("go door", "Obvious exits", "You must be standing to do that")
+    when "You must be standing to do that"
+      bput('stand', 'You stand back up')
+      bput("go door", "Obvious exits")
+    end
+    bput("get prize", "The pumpkin farmer beams happily at you", "The pumpkin farmer grins at you")
+    pause
+  end
+
+  def get_pumpkin
+    @true_pumpkin = false
+    case bput('get pumpkin', 'You help yourself', 'You pick up')
+    when 'You pick up'
+      if @fake_pumpkins.any? { |item| Regexp.new(item) =~ right_hand }
+        stow_hands
+      else
+        @true_pumpkin = true
+      end
+    else
+      @true_pumpkin = true
+    end
+  end
+
+  def check_prize
+    if @trash_items.any? { |item| Regexp.new(item) =~ right_hand }
+      bput("put my #{righthand} in bucket", "You put", "You drop")
```

Comment:
❓ Should this be `#{right_hand}` (note the underscore)?

Reply:
@bmcneff @rcuhljr I think this is the last thing holding up merging. Does `righthand` work as well as `right_hand`?

Reply:
#{righthand} is a builtin alias to checkright:

brian@bmsr-dr:~$ grep ":righthand" lich/lich.rbw 
alias :righthand? :checkright
alias :righthand :checkright
brian@bmsr-dr:~$

It was an easy way to simply reference the noun rather than the full item.  I can switch it back to checkright if you prefer.

Reply:
As long as it works, I won't hold up merging for it.